crates/ruff/src/rules/ruff/rules/confusables.rs

```diff
@@ -1,1590 +1,1589 @@
-    120618u32 => 79,
-    120620u32 => 80,
-    120623u32 => 84,
-    120624u32 => 89,
-    120626u32 => 88,
-    120630u32 => 97,
-    120632u32 => 121,
-    120638u32 => 105,
-    120642u32 => 118,
-    120644u32 => 111,
-    120646u32 => 112,
-    120648u32 => 111,
-    120650u32 => 117,
-    120660u32 => 112,
-    120662u32 => 65,
-    120663u32 => 66,
-    120666u32 => 69,
-    120667u32 => 90,
-    120668u32 => 72,
-    120670u32 => 73,
-    120671u32 => 75,
-    120673u32 => 77,
-    120674u32 => 78,
-    120676u32 => 79,
-    120678u32 => 80,
-    120681u32 => 84,
-    120682u32 => 89,
-    120684u32 => 88,
-    120688u32 => 97,
-    120690u32 => 121,
-    120696u32 => 105,
-    120700u32 => 118,
-    120702u32 => 111,
-    120704u32 => 112,
-    120706u32 => 111,
-    120708u32 => 117,
-    120718u32 => 112,
-    120720u32 => 65,
-    120721u32 => 66,
-    120724u32 => 69,
-    120725u32 => 90,
-    120726u32 => 72,
-    120728u32 => 73,
-    120729u32 => 75,
-    120731u32 => 77,
-    120732u32 => 78,
-    120734u32 => 79,
-    120736u32 => 80,
-    120739u32 => 84,
-    120740u32 => 89,
-    120742u32 => 88,
-    120746u32 => 97,
-    120748u32 => 121,
-    120754u32 => 105,
-    120758u32 => 118,
-    120760u32 => 111,
-    120762u32 => 112,
-    120764u32 => 111,
-    120766u32 => 117,
-    120776u32 => 112,
-    120778u32 => 70,
-    120782u32 => 79,
-    120783u32 => 73,
-    120784u32 => 50,
-    120785u32 => 51,
-    120786u32 => 52,
-    120787u32 => 53,
-    120788u32 => 54,
-    120789u32 => 55,
-    120790u32 => 56,
-    120791u32 => 57,
-    120792u32 => 79,
-    120793u32 => 73,
-    120794u32 => 50,
-    120795u32 => 51,
-    120796u32 => 52,
-    120797u32 => 53,
-    120798u32 => 54,
-    120799u32 => 55,
-    120800u32 => 56,
-    120801u32 => 57,
-    120802u32 => 79,
-    120803u32 => 73,
-    120804u32 => 50,
-    120805u32 => 51,
-    120806u32 => 52,
-    120807u32 => 53,
-    120808u32 => 54,
-    120809u32 => 55,
-    120810u32 => 56,
-    120811u32 => 57,
-    120812u32 => 79,
-    120813u32 => 73,
-    120814u32 => 50,
-    120815u32 => 51,
-    120816u32 => 52,
-    120817u32 => 53,
-    120818u32 => 54,
-    120819u32 => 55,
-    120820u32 => 56,
-    120821u32 => 57,
-    120822u32 => 79,
-    120823u32 => 73,
-    120824u32 => 50,
-    120825u32 => 51,
-    120826u32 => 52,
-    120827u32 => 53,
-    120828u32 => 54,
-    120829u32 => 55,
-    120830u32 => 56,
-    120831u32 => 57,
-    125127u32 => 108,
-    125131u32 => 56,
-    126464u32 => 108,
-    126500u32 => 111,
-    126564u32 => 111,
-    126592u32 => 108,
-    126596u32 => 111,
-    128844u32 => 67,
-    128872u32 => 84,
-    130032u32 => 79,
-    130033u32 => 73,
-    130034u32 => 50,
-    130035u32 => 51,
-    130036u32 => 52,
-    130037u32 => 53,
-    130038u32 => 54,
-    130039u32 => 55,
-    130040u32 => 56,
-    130041u32 => 57,
-};
+pub(crate) fn confusable(c: u32) -> Option<u8> {
+    match c {
+        160u32 => Some(32),
```

Comment:
Can we use `char` instead of `u32`. I have no idea what I'm looking at :sweat_smile:

Reply:
Yeah easily. I assume the LHS should remain u32 though?

Reply:
What would be the reason to prefer the left side to be a `u32` instead of a `char`? I could also think of using the unicode names, but that's probably harder (maybe rustfmt does that for you)

Reply:
We end up with a bunch of confusing Unicode characters in the source, some of which my editor can't even render I think, some of which are bidirectional...